prog.go

```diff
@@ -107,6 +109,28 @@ func (ps *ProgramSpec) Tag() (string, error) {
 	return ps.Instructions.Tag(internal.NativeEndian)
 }
 
+// SetAttachTarget configures the program to be attached to the optionally
+// specified function in the given program.
+//
+// If the function is not specified, it must have been set via other means
+// (such as loaded from the ELF file via the section name).
+func (ps *ProgramSpec) SetAttachTarget(program *Program, function string) error {
```

Comment:
What's the rational to have a dedicated setter? Both `AttachTarget` and `AttachTo` are accessible to the user, so I'd prefer to not have this function.

Reply:
Since (1) doesn't really apply here, and (3) can't enforce the variants because the members are public, I've removed the function. (For people in situation (2), they're either using old code, in which case they're probably not using freplace and this doesn't apply, or they're opting to not do things "the easy way" anyway, so an extra line of setup isn't a big deal.)